macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -26,7 +26,6 @@
     },
     'time_format': '%Y-%m-%d %H:%M:%S',
     'long_string_size': 16384,
-    'collect_model_sql': true,
```

Comment:
Why is this configurable?

Reply:
last resort if the length limitations didn't help